Bug 819304 - "Clear all" button should have a hit state

durwasa-chakraborty opened the pull request:
The notification tray and the button that reads Clear all now has a hit state.

albertopq committed:
Merge pull request #31709 from mikehenrty/bug-1202256-gaia-toast

Bug 1202256 - Use gaia-toast web component for System Banner r=albertopq

Commit:
Bug 1223661 - Update gaia-taskenv to tester-0.4.4 base image, update world to run node 4.2.2. r=garndt, gaye

autolander commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7c3e6fd5be69a0c580c436f355ec8298a3d3e002

Commit:
Bug 1212402 - [TV 2.5] Apply visual spec to the TV remote control app

sfoster committed:
Bug 1223578 - Refactor CSS for bidi best practices. r=salva

evanxd committed:
Bug 1214908 - Change audio channel type of unlock audio as system r=snowmantw

gabrielesvelto committed:
Bug 1224540 - Adjust the SIM indicator color and position in the incoming call container r=thills

punamdahiya committed:
Bug 1219657 - [Gallery] Implement delete multiple images Gij test

rexboy7 committed:
Bug 1225062 - [TV][FTE] Catch focus on mouse click

Commit:
Bug 1223420 - Avoid creating intermediate array in DataAdapters, r=weilonge

ferjm committed:
Bug 1224204 - 'Get Started' button does nothing after a Server Unreachable error. r=michielbdejong,yifan

scottwu committed:
Bug 1219693 - Improved existing test cases for USB settings

Commit:
Bug 1204830 - check meta/global engines, r=ferjm

mcav committed:
Bug 1197989 - Intermittent apps/clock/test/marionette/hour_format_test.js | Hour format 24 hour format Fire an alarm.

KevinGrandon committed:
Bug 1225765 - [Gallery] Port crop aspect ratio to use gaia-menu r=pdahiya

nojunpark committed:
Bug 1226358 - Imagecompare: test_settings_NC_RTL.py fails to tap caller_id

danhuang1202 committed:
Bug 1225431 - revice l10 String to fxsync-sign-out and fxsync-confirm-sign-out. r=seanlee

danhuang1202 committed:
Bug 1226078 - revise title in fxaccount enter password page in TV browser. r=seanlee

evelynhung committed:
Bug 1226573 - manually handle focus movement between promotion list and app list.

1. remove promotion list from app's spatialNavigator control.
2. set a flag to indicate focusing area when focus moves between promotion list and app list.
3. in each area, call its spatial navigator to move focus.

Commit:
Bug 1224065 - Migrate FxA to use modern L10n API

Commit:
NO BUG - Bumping revision to see if it helps tc-vcs get unjammed. a=nochange

Commit:
NO BUG - Remove useless parameter from git checkout function in entrypoint of gaia-taskenv in hopes of making tc-vcs happy.

Commit:
Revert "Merge pull request #33310 from nullaus/master"

This reverts commit 56d0ee41ae198aa8b9ac585ce8d2ee765794ec59, reversing
changes made to 93c139915f99b5770a23e3e3591c9ffcf6888bf8.

Commit:
Revert "Merge pull request #33309 from nullaus/master"t push

This reverts commit 93c139915f99b5770a23e3e3591c9ffcf6888bf8, reversing
changes made to b93f975cc713364b657f125d116f1d98bc47a739.

KevinGrandon committed:
Bug 1225424 - Restore missing string, required for the pick activity when no videos are present

durwasa-chakraborty committed:
Bug 1156469 - Error message that says App Install error: DOWNLOAD_CANCELED

evelynhung committed:
Revert "Merge pull request #33297 from evelynhung/issue-1226573"

This reverts commit ab4c01c2799ab10d41bd937426a4e93907b44304, reversing
changes made to bae13c9ac6a91beecd7c94384e2aef25ed1a3214.

durwasa-chakraborty committed:
Merge branch 'master' of https://github.com/durwasa-chakraborty/gaia

durwasa-chakraborty committed:
Merge branch 'install-cancelled'
